tests/test-cli-coverage-gaps.sh

```
@@ -0,0 +1,105 @@
+zipper --loadfile-only --count 5 --output-path "$TEST_OUTPUT_DIR/multi_delim" \
+    --multi-delim "char:;" --column-profile standard > /dev/null 2>&1
+props_file=$(find "$TEST_OUTPUT_DIR/multi_delim" -name "*_properties.json" -print -quit)
+if grep -q '"multiValue": "char:;"' "$props_file"; then
+    pass "--multi-delim recorded in properties"
+else
+    fail "--multi-delim recorded in properties"
+fi
+
+print_info "Test: --nested-delim recorded in properties"
+zipper --loadfile-only --count 5 --output-path "$TEST_OUTPUT_DIR/nested_delim" \
+    --nested-delim 'char:\' --column-profile standard > /dev/null 2>&1
```

Shellcheck (0.11.0)

[info] Want to escape a single quote? echo 'This is how it'\''s done'.

(SC1003)